python/ambassador/utils.py

```diff
@@ -59,7 +59,7 @@
 yaml_logged_dumper = False
 
 
-def parse_yaml(serialization: str, **kwargs) -> Any:
+def parse_yaml(serialization: str) -> Any:
```

Comment:
I removed the kwargs because they weren't being passed to the underlying parser and seemed to be confusingly misused in the resource fetcher.